libdnf.spec

```diff
@@ -35,6 +35,7 @@ BuildRequires:  gcc
 BuildRequires:  libsolv-devel >= %{libsolv_version}
 BuildRequires:  pkgconfig(librepo)
 BuildRequires:  pkgconfig(check)
+BuildRequires:  redhat-rpm-config
```

Comment:
@Conan-Kudo 
When building from fedora 25 docker image without `redhat-rpm-config` I am getting this: https://paste.fedoraproject.org/537115/